actors/core/src/main/java/com/ea/orbit/actors/runtime/OrbitActor.java

```diff
@@ -120,9 +121,9 @@ protected T state()
      *
      * @return a completion promise
      */
-    protected Task<Boolean> readState()
+    protected Task<Void> readState()
```

Comment:
Cool, you removed the unused boolean. I wonder if it shouldn't signal if the data was found or not.